chain/chain.go

```diff
@@ -226,21 +250,27 @@ func (g *Genesis) UnmarshalJSON(data []byte) error {
 
 // GenesisAccount is an account in the state of the genesis block.
 type GenesisAccount struct {
-	Code          []byte                    `json:"code,omitempty"`
-	Storage       map[types.Hash]types.Hash `json:"storage,omitempty"`
-	Balance       *big.Int                  `json:"balance,omitempty"`
-	StakedBalance *big.Int                  `json:"stakedBalance,omitempty"` // Staking implementation
-	Nonce         uint64                    `json:"nonce,omitempty"`
-	PrivateKey    []byte                    `json:"secretKey,omitempty"` // for tests
+	Code       []byte                    `json:"code,omitempty"`
+	Storage    map[types.Hash]types.Hash `json:"storage,omitempty"`
+	Balance    *big.Int                  `json:"balance,omitempty"`
+	Nonce      uint64                    `json:"nonce,omitempty"`
+	PrivateKey []byte                    `json:"secretKey,omitempty"` // for tests
```

Comment:
It's just to separate it out from the staking implementation and keep it uniform, since it's not present in any other _legacy_ objects.
We added an analog way to `Alloc` -> `AllocStake` to keep track of prestaked balances